### PR TITLE
AgentTaskFlow

### PR DESCRIPTION
**Pull request by mauryaakash021**

### CODE REVIEW

File: src/test/java/com/bharatpe/steps/AgentTaskSteps.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    public void agentCompletingLoanTask() {
+        agentTaskPage.agentCompletingLoanTask();
+    }
+
+    @When("cancel visit")
+    public void cancelVisit() {
+        agentTaskPage.cancelVisit();
+    }
+
+
```
Comment (jitendrapisal55) [inline src/test/java/com/bharatpe/steps/AgentTaskSteps.java:41]: @mauryaakash021  remove extra spaces

File: src/test/java/com/bharatpe/steps/AgentTaskSteps.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
+package com.bharatpe.steps;
+
+import com.bharatpe.pages.AgentTaskPage;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class AgentTaskSteps {
+    public AgentTaskPage
+            agentTaskPage
+            ;
```
Comment (jitendrapisal55) [inline src/test/java/com/bharatpe/steps/AgentTaskSteps.java:10]: @mauryaakash021  align your code. 
e.g - Open the file that you want to align.
Press  Command + option + L

File: src/test/resources/features/agentTask.feature [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@agent-tasks
+Business Need: agent task flow
+
+  Background: verify created list of task on home page
+    Then agent creates the task
+    Then agent confirms the task
```
Comment (jitendrapisal55) [inline src/test/resources/features/agentTask.feature:6]: @mauryaakash021  remove this from background.
And create method like this in AgentTaskSteps

```
static boolean runOnlyOnce = true;

    @Before("@agent-tasks")
    public static void createTaskInit() {
        if (runOnlyOnce) {
            // call the method that are in the background here.
        }
        runOnlyOnce = false;
    }
```

**Review by jitendrapisal55 — CHANGES_REQUESTED** [review]
@mauryaakash021  Please check the comments

File: src/test/java/com/bharatpe/steps/AgentTaskSteps.java (hunk 2 above)
Comment (mauryaakash021) [reply src/test/java/com/bharatpe/steps/AgentTaskSteps.java:10]: @jitendrapisal55     i Will take care in future.

File: src/test/resources/features/agentTask.feature (hunk 3 above)
Comment (jitendrapisal55) [reply src/test/resources/features/agentTask.feature]: @mauryaakash021  work on it immediately

Comment (mauryaakash021) [reply src/test/resources/features/agentTask.feature]: @jitendrapisal55   i have resolved it and pushed also.

**Review by jitendrapisal55 — APPROVED** [review]

**Pull request merged**